practice-app/server/src/api/controllers/event/createEventEndpoint.js

```diff
@@ -0,0 +1,27 @@
+import { createEvent } from "./createEvent.js";
+
+export default async (req, res) => {
+    try {
+        const {resultMessage, event} = await createEvent(req.body.title, req.body.date, req.body.location, req.body.host_id, req.body.lesson_id);
+        let status = 200;
+        console.log(resultMessage);
+        return res.status(status).json({"resultMessage": resultMessage, "event": event});
+    }catch(err) {
+        let status = 500;
+        if(err.message == "\"title\" length must be at least 3 characters long")
```

Comment:
Normally, returning the status codes in the places where we detected the errors is the preferred approach. However, this is OK for now for the practical purposes. So, you can just resolve this without taking any action.